sphinx/builders/latex/__init__.py

```diff
@@ -403,15 +402,15 @@ def validate_config_values(app, config):
     # type: (Sphinx, Config) -> None
     for document in config.latex_documents:
         try:
-            text_type(document[2])
+            str(document[2])
```

Comment:
This and next call try to check the member of `latex_documents` contains unicode chars or not.
But this validation has been finished their role because py3 syntax does not generate bytes object in standard use.

I think removing both block is better. What do you think?

Reply:
Done in #5813